### DISCUSSION TITLE
Change the name of WebUI project to just Web

### DISCUSSION THREAD

**Discussion by hesperanca** [jasontaylordev/CleanArchitecture#526] [Ideas]
Hi all,

Is it possible to rename the WebUI project to just Web? The reason for this is that it would cover both scenarios where we are creating a simple REST API project with no UI and a project that has UI elements. I've tried to rename it but it becomes a pain because we need to change all the references, namespaces, etc etc.

Just an idea.

Thanks for the great work btw.

**Comment by Brhav** [comment]
Seconding this, same reason
